### ISSUE TITLE
Nieprawidłowe działanie mechanizmu layers

### ISSUE THREAD

**Issue by Zwierzu** [epforgpl/_mojePanstwo-Portal#806]
Zgodnie z dokumentacją udostępnioną pod adresem "https://api-v3.mojepanstwo.pl/dane/" api powinno dla poszczególnych "datasets" udostępniać listę "layers". Tak się nie dzieje. 

Przykładowe wywołanie: "https://api-v3.mojepanstwo.pl/dane/krs_podmioty/10186" zwraca w liście layers tylko kilka wpisów ("dataset", "channels", "page", "subscribers"). W rzeczywistości lista wpisów powinna być znacznie szersza (np.: "reprezentacja", "graph", "firmy" i tak dalej) tak jak to udokumentowano pobieżnie na stronie: "https://mojepanstwo.pl/api/krs".

Taki sam problem występuje dla wszystkich "datasets" jakie testowałem.

**Comment by DontEatRice** [comment]
Tak, miałem pisać o tym samym, moim zdaniem jest to chyba najciekawsza opcja, która dałaby dużo możliwości
